### PR TITLE
Remaps the Shetland

### PR DESCRIPTION
**Pull request by Apogee-dev**
## About The Pull Request
faroe
faroe_areas

Brings the Shetland up to modern mapping standards, makes the layout less garbage, adds some amenities and RP facilities, and rebalances the ship's mechanical features.

- RnD and ORM removed
- TEG added
- Weapons and armor rebalanced
- Massive aesthetic improvements

## Why It's Good For The Game

Shetland is the oldest ship we have left in the game and despite getting a couple updates along the way it really shows. It was either update it or kill it, and I think the Shetland still has good potential and is worth saving.

## Changelog

:cl:
tweak: Remapped the Shetland
balance: Removed RnD and ORM from Shetland
/:cl:

### CODE REVIEW

**Comment by meemofcourse** [comment]
sus

**Comment by Cadunkus** [comment]
Aw now there's no empty space to do engineering projects. Can't even fit a supermatter engine in the back anymore the Shetland was pretty much the only ship that could pull that off without a huge remodeling.

File: code/modules/clothing/outfits/factions/independent.dm [hunk 1]
```diff
@@ -91,6 +91,11 @@
 
 	chameleon_extras = list(/obj/item/gun/energy/e_gun, /obj/item/stamp/captain)
 
+/datum/outfit/job/independent/captain/cheap //for Miskilamo ships
+	name = "Independent - Captain (Low Budget)"
+	gloves = /obj/item/clothing/gloves/color/white //poverty gloves
```
Comment (Latentish) [inline code/modules/clothing/outfits/factions/independent.dm:96]: Hilarious.

File: shiptest.dme [hunk 2]
```diff
@@ -744,8 +744,8 @@
 #include "code\datums\votes\_vote_datum.dm"
 #include "code\datums\votes\custom_vote.dm"
 #include "code\datums\votes\restart_vote.dm"
-#include "code\datums\weather\weather.dm"
 #include "code\datums\votes\transfer_vote.dm"
+#include "code\datums\weather\weather.dm"
```
Comment (Latentish) [inline shiptest.dme]: This is an irrelevant change. Swap this back so we don't have a DME edit tag here.

**Review by Latentish — DISMISSED** [review]

**Comment by Latentish** [comment]
Finally some good fucking remap. I'll test this on live as soon as you aren't PRing a DME change (rip all other tm's that add a new file)

**Comment by Cadunkus** [comment]
I don't see a fax machine in the .dmm file with ctrl+f.

**Comment by Apogee-dev** [comment]
> I don't see a fax machine in the .dmm file with ctrl+f.

Indie ships don’t get faxes. I was in fact asked to remove one in an early draft.

**Comment by Latentish** [comment]
Tested this on live, seems to be pretty much good? So because I don't have much to say I'm going to pass this review off to the other mapties.

**Review by meemofcourse — APPROVED** [review]
yeah

**Comment by tiramisuapimancer** [comment]
this is a good remap, but the fact that it isnt the skeld anymore kind of defeats the point doesnt it

**Comment by Latentish** [comment]
> this is a good remap, but the fact that it isnt the skeld anymore kind of defeats the point doesnt it

No. We don't need a n among us reference. Also the old layout was the thing that sparked the ship cute.



This ship is good for a merge smile.

**Pull request merged**